### DISCUSSION TITLE
Open-Shell removing duplicate program listings

### DISCUSSION THREAD

**Discussion by laynelab** [Open-Shell/Open-Shell-Menu#1288] [Q&A] [answered]
In the start menu are several dual listings;
example:
Autorun Organizer
Autorun Organizer(2)
I tried many things and need to learn how to remove the second item in the listing. Can someone help?

**Comment by ge0rdi** [comment, marked as answer]
You can create `Open-Shell` log using this [guide](https://github.com/Open-Shell/Open-Shell-Menu/wiki/How-to-obtain-advanced-Open-Shell-Menu-log) to see where the item came from.
Eventually attach it here so that we can have a look.

**Comment by laynelab** [comment]
Here is Open-SHell log:
[StartMenuLog.txt](https://github.com/Open-Shell/Open-Shell-Menu/files/10430137/StartMenuLog.txt)

**Reply by ge0rdi** [reply]
You have the shortcut placed in start menu folder:
`C:\Users\ae1nk\AppData\Roaming\Microsoft\Windows\Start Menu\Autorun Organizer (2).lnk`

Remove it and it will not show in Start menu anymore.

**Comment by laynelab** [comment]
Thanks a million!
Layne AE1N
